<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="showText">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>441</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpParse">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>650</y>
      <width>441</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Выбор слов</string>
    </property>
    <widget class="QRadioButton" name="radioAll">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>106</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Все</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioNum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Содержащие цифры</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioMail">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Содержащие e-mail</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnParse">
     <property name="geometry">
      <rect>
       <x>193</x>
       <y>37</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Parse!</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpSearch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>471</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Поиск</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Введите искомое слово</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineSearch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>271</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="listSearch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>271</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSearch">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkR1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>90</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Раздел 1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkR2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>90</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Раздел 2</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>527</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Сброс</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>650</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listR1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listR2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Раздел 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Раздел 2</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboR1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>281</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>По длине (возр.)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>По длине (убыв.)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>По алфавиту (возр.)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>По алфавиту (убыв.)</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboR2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>281</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>По длине (возр.)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>По длине (убыв.)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>По алфавиту (возр.)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>По алфавиту (убыв.)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnLtoRsingle">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRtoLsingle">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLtoRmultiple">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRtoLmultiple">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSortR1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearR1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSortR2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearR2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>430</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
